Description: Do not link to KDE3Support, only Qt3Support is needed
Author: Luigi Toscano <luigi.toscano@tiscali.it>
Forwarded: no
Last-Update: 2010-04-12 
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -59,7 +59,7 @@
 kde4_add_library(basketcommon SHARED ${basketcommon_LIB_SRCS})
 
 target_link_libraries(basketcommon
-  ${KDE4_KDE3SUPPORT_LIBS} # Remove me
+  ${QT_QT3SUPPORT_LIBRARY}
   ${KDE4_KIO_LIBS}
   ${KDE4_KDEUI_LIBS}
   ${KDE4_KPARTS_LIBS}
@@ -82,7 +82,7 @@
 
 kde4_add_executable(basket ${basket_SRCS})
 target_link_libraries(basket
-  ${KDE4_KDE3SUPPORT_LIBS} # Remove me as well
+  ${QT_QT3SUPPORT_LIBRARY}
   ${KDE4_KUTILS_LIBS}
   ${KDE4_KFILE_LIBS}
   ${KDE4_PHONON_LIBRARY}
